src/containers/ExternalKeyInput/ExternalKeyInput.test.tsx

```diff
@@ -0,0 +1,30 @@
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { render, screen } from "../../testUtils";
+import { ExternalKeyInput } from "./ExternalKeyInput";
+
+describe("ExternalKeyInput", () => {
+  test("Should post correct external key to rapid assessment form", async () => {
+    const { history } = render(<ExternalKeyInput />);
```

Comment:
@johnhok Adding `history` to our custom render was a stroke of genius - really simplifies checking paths and params!!